lib/internal/url.js

```diff
@@ -1365,27 +1366,54 @@ const backslashRegEx = /\\/g;
 const newlineRegEx = /\n/g;
 const carriageReturnRegEx = /\r/g;
 const tabRegEx = /\t/g;
+
+function encodePathChars(filepath) {
+  if (filepath.includes('%'))
+    filepath = filepath.replace(percentRegEx, '%25');
+  // In posix, "/" is a valid character in paths
```

Comment:
```suggestion
  // In posix, "\" is a valid character in paths
```

Reply:
Ah, good catch. I just moved this comment, btw, when I refactored out the function. Will fix.